docs/hardware.rst

```diff
@@ -82,10 +82,10 @@ Workstations
   <https://tails.boum.org/news/Tails_3.0_will_require_a_64-bit_processor/index.en.html>`_.
 
 These components are necessary to do the initial installation of
-SecureDrop and to process submissions using the airgapped workflow.
+SecureDrop and to process submissions using the air-gapped workflow.
```

Comment:
I'm against this change simply because it's a stylistic deviation that breaks consistency throughout the docs. For instance, prior to this PR, the hyphenated version was very much in the minority:

```
grep -riP 'air.?gap' -o | cut -d: -f2  | sort | uniq -c
     24 airgap
      9 air-gap
      1 Airgap
```

Including the changes in this PR, the situation is even more fractious: 

```
     19 airgap
      4 air gap
     10 air-gap
      1 Airgap
```

Therefore I vote we stick with the non-hyphenated, non-whitespace version.